apps/ledger-live-desktop/src/renderer/screens/account/AccountHeaderActions.jsx

```diff
@@ -277,7 +310,9 @@ const AccountHeaderActions = ({ account, parentAccount, openModal }: Props) => {
       : []),
   ];
 
-  const BuyHeader = <BuyActionDefault onClick={onBuy} />;
+  const BuyHeader = <BuyActionDefault onClick={() => onBuySell("buy")} />;
+
+  const SellHeader = <SellActionDefault onClick={() => onBuySell("sell")} />;
 
   const SwapHeader = <SwapActionDefault onClick={onSwap} />;
```

Comment:
these variables should start with a lowercase. It's a React$Node, not a React$ComponentType.